packages/flutter/lib/src/material/grid_tile_bar.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'icon_theme.dart';
+import 'icon_theme_data.dart';
+import 'typography.dart';
+
+
+/// Typically used to stack a one or two line header or footer on a Grid tile.
+/// The layout is based on the "Grid Lists" section of the Material Design spec:
+/// https://www.google.com/design/spec/components/grid-lists.html#grid-lists-specs
+/// For a one-line header specify title and to add a second line specify caption.
+/// Use left or right to add an icon.
+class GridTileBar extends StatelessComponent {
```

Comment:
Good point. Since the GridTileBar can serve as a tile header or footer, maybe a separate GridTile widget would eliminate the Stack boilerplate:

```
class GridTile {
  GridTile({ Key key, this.header, this.footer, this.child }) : super(key: key);

  final Widget header;
  final Widget footer;
  final Widget child;

  ...
}

new GridTile(
  header: new GridTileBar(
    title: new Text('the footer appears on the bottom of the tile')
  ),
  child: new AssetImage(
     ...
  )
)
```